.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\97254\\AppData\\Local\\Microsoft\\WindowsApps\\python.exe"
+}
```

Comment:
this file shouldn't be in the repo.
You can exclude it by adding the .vscode folder to the .gitignore file